docs/jupyter/altair.ipynb

```diff
@@ -0,0 +1,217 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Demonstrating Altair Modin Interoperability\n",
+    "### Each visualization is created with a Modin and then pandas dataframe for comparison."
```

Comment:
I only see that you use a Modin df and get the error. Should we clarify the title that Modin is not interoperable with altair?

Reply:
That's a good clarification - I've added this to the subheading